composables/useAmount.ts

```diff
@@ -5,15 +5,15 @@ export default function (
   tokenAmount: ComputedRef<number | string | undefined>,
   tokenDecimals: ComputedRef<number>,
   chainSymbol: ComputedRef<string>,
-  roundBy?: ComputedRef<Prefix | number>,
+  { roundBy, withBlank = false }: { roundBy?: ComputedRef<Prefix | number>, withBlank?: boolean } = {},
 ) {
   const { getCurrentTokenValue } = useFiatStore()
 
   const amountFormatted = computed(() => {
     const amount = tokenAmount.value
       ? formatAmountWithRound(tokenAmount.value, tokenDecimals.value, roundBy?.value)
       : 0
-    return `${amount} ${chainSymbol.value}`
+    return ((!amount || amount === '0') && withBlank) ? '--' : `${amount} ${chainSymbol.value}`
```

Comment:
```suggestion
    return !Boolean(Number(amount)) && withBlank ? '--' : `${amount} ${chainSymbol.value}`
```

make the conditional checking a util inside the composable and also uses it inside `amountUsd`



````

const amountWithBlank = (amount) => 

````